worldcup/README.md

```diff
@@ -22,21 +22,111 @@ Mexico: 0.1% chance of winning
 
 ## Background
 
-TODO
+* In soccer, teams are given [FIFA Ratings](https://en.wikipedia.org/wiki/FIFA_World_Rankings#Current_calculation_method), which are numerical values representing the team's relative skill levels. Higher FIFA ratings indicate better previous game results, and we can determine the probability that a team wins a matchup if we have both teams' FIFA ratings.
```

Comment:
Even though students don't have to write it, maybe give the formula here so it's a little more real?